libs/src/sdk/oauth/OAuth.ts

```diff
@@ -140,7 +144,7 @@ export class OAuth {
     )
   }
 
-  login() {
+  async login() {
```

Comment:
What code calls this function? Is it set up to work properly with `async` (i.e. wait for the promise returned here).